composer.json

```diff
@@ -29,10 +29,10 @@
     "php":                                ">=5.6",
     "guzzlehttp/guzzle":                  "~6.0",
     "eightpoints/guzzle-wsse-middleware": "~4.0",
-    "symfony/dependency-injection":       "~2.3|~3.0",
-    "symfony/expression-language":        "~2.3|~3.0",
-    "symfony/event-dispatcher":           "~2.3|~3.0",
-    "symfony/http-kernel":                "~2.3|~3.0",
+    "symfony/dependency-injection":       "~3.2",
```

Comment:
with this change we only support symfony >= 3.2, right? I would say, this is a breaking change and needs a new major version than. README.md should also be updated.

Reply:
Yep, it will support only symfony >= 3.2, but next version is Guzzle Bundle 7 and it's already released :slightly_frowning_face: I think it's more like fix and as I wrote, who is working with SF < 3.2 symfony will not receive this fix, but who work with symfony >= 3.2 and wont still to use v6 of bundle for some reason then they will receive this fix. I think it's more like fix/patch. What do you think? If ok then I will fix travis and readme.

Reply:
Alright, go for it! 👍